source/funkin/play/PlayState.hx

```diff
@@ -891,34 +891,6 @@ class PlayState extends MusicBeatSubState
       needsReset = false;
     }
 
-    // Update the conductor.
```

Comment:
The conductor should definitely be processed before the note hit so that the song position used when calculating note hits is as accurate as possible. If you don't update the conductor first, the game will be comparing the time the note was hit with the time the song was at on the PREVIOUS frame.